worlds/no_roof_small_warehouse.world

```diff
@@ -224,7 +224,7 @@
       <specular>0.2 0.2 0.2 1</specular>
       <attenuation>
         <range>80</range>
-        <constant>0.01</constant>
+        <constant>0.3</constant>
```

Comment:
This attenuation value is tuned for CPU or GPU?

Reply:
yep, these values work well for both with/without GPU